src/Jaeger.Thrift/Senders/Internal/ThriftUdpClientTransport.cs

```diff
@@ -9,7 +9,7 @@ namespace Jaeger.Thrift.Senders.Internal
 {
     public class ThriftUdpClientTransport : TTransport
     {
-        public const int MaxPacketSize = 65000; // TODO !!! Not yet used.
+        public const int MaxPacketSize = 65000;
```

Comment:
no other changes in this file, is it still unused?

Reply:
Just found the left-over comment and saw that was wrong. It is in use as it will be given to ThriftSenderBase through UdpSender where it‘s used.